packages/back-end/src/models/ApiKeyModel.ts

```diff
@@ -10,6 +10,9 @@ const apiKeySchema = new mongoose.Schema({
   description: String,
   organization: String,
   dateCreated: Date,
+  encryptFeatures: Boolean,
+  encryptionPublicKey: String,
```

Comment:
@jdorn I've looked and looked and tried a bunch of different things, but I can't find a solution. Currently, when a user creates a new encrypted API key, we're generating the public/private keypair, and then converting them (via a pretty lengthy process) into strings in order to save them in Mongo. However, it feels unneccessary to do that with the publicKey - do you know of a way to save an `ArrayBuffer` to MongoDB? This would remove a lot of work of having to convert it to a string to save to the DB, and then convert it back into an `ArrayBuffer` in order to actually encrypt the features list.